tests/test_all_simple_paths.py

```diff
@@ -0,0 +1,347 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import unittest
+
+import retworkx
+
+
+class TestDAGAllSimplePaths(unittest.TestCase):
+    def test_all_simple_paths(self):
+        dag = retworkx.PyDAG()
+        for i in range(6):
+            dag.add_node(i)
+        edges = [
```

Comment:
maybe use a setup func and set this to `self.edges` since it looks like they're all the same (and would be helpful for reviewing if I can assume it's the same graph)

Reply:
Fixed in: https://github.com/Qiskit/retworkx/pull/85/commits/0cc66afae85e70b67096306e4900137a1476a166